.github/workflows/backend.yml

```diff
@@ -329,10 +329,6 @@ jobs:
           snuba: true
           kafka: true
 
-      - name: Run snuba test (${{ steps.setup.outputs.matrix-instance-number }} of ${{ steps.setup.outputs.matrix-instance-total }})
-        run: |
-          make test-snuba
```

Comment:
you can actually delete this whole section in one push -- the required checks is listed at the bottom so we don't have to juggle github config any more

Reply:
Oh cool, we should do this in our other repos too. Is there anything else to be changed here? Afaict it's just these 3 lines which are already gone.

Reply:
there's a whole `snuba:` block and then below there's a `snuba` block below